config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -1467,6 +1467,28 @@ spec:
                         type: string
                     type: object
                 type: object
+              rsyslog_resource_requirements:
+               description: Resource requirements for the rsyslog container   
+               properties:
+                 requests: 
+                    properties:
+                      cpu:
+                        type: string
+                      memory:
+                        type: string
+                      storage:
+                        type: string
+                    type: object
+                 limits:
```

Comment:
The indentation seems to be wrong for this whole block (I think from ln1482 --> 1491).